cmd/compute-domain-kubelet-plugin/main.go

```diff
@@ -34,9 +34,10 @@ import (
 const (
 	DriverName = "compute-domain.nvidia.com"
 
-	PluginRegistrationPath     = "/var/lib/kubelet/plugins_registry/" + DriverName + ".sock"
+	// Plugin registration socket filename and path have a sane default set by
+	// kubeletplugin/draplugin.go (at the time of writing that is
+	// "/var/lib/kubelet/plugins_registry/" + driverName + "-reg.sock").
```

Comment:
I can remove the code comment.

But yeah, the relevant callout is that the dra lib new provides sane defaults and we don't need to define socket paths in our driver implementation.

Reply:
Is this something to add to the README at least? (is the current socket mentioned there at all)?

One thing that I have seen is that it may be useful to be able to override this socket for some testing scenarios, but I think adding it at a later stage if required makes more sense.

Reply:
> Just wanted to make sure I communicate this explicitly: the DRA lib now provides sane defaults and we don't need to define socket paths in our driver implementation anymore (unless we have strong reason).

Every DRA driver should have a configuration option or parameter for it. Otherwise it is unusable on non-standard clusters.

Reply:
> Every DRA driver should have a configuration option or parameter for it. Otherwise it is unusable on non-standard clusters.

Ha. Thank you for the feedback, Patrick!

I created https://github.com/NVIDIA/k8s-dra-driver-gpu/issues/339 to track this.

> Is this something to add to the README at least?

Yes. Let's add config params, and then properly document the configuration interface. (independent of this PR)

Reply:
Code comment can be removed

Reply:
done